Missing Dependency react-highlight-words

> ./node_modules/chakra-ui-autocomplete/dist/chakra-ui-autocomplete.cjs.development.js
Module not found: Can't resolve 'react-highlight-words' in '[my-project-path]\node_modules\chakra-ui-autocomplete\dist'

I believe this needs to be added as a dependency. Just working through to get this working, and this was one error that occurred.

--- a/package.json
+++ b/package.json
@@ -59,10 +59,8 @@
 		"prettier": "^2.0.4",
 		"react": "^16.13.1",
 		"react-dom": "^16.13.1",
-		"react-highlight-words": "^0.16.0",
 		"react-scripts": "^3.4.1",
 		"react-test-renderer": "^17.0.1",
-		"react-use": "^15.3.4",
 		"rimraf": "^3.0.2",
 		"tsdx": "^0.14.1",
 		"tslib": "^2.0.3",
@@ -88,6 +86,8 @@
 		"url": "git+https://github.com/koolamusic/chakra-ui-autocomplete.git"
 	},
 	"dependencies": {
-		"downshift": "^6.0.6"
+		"downshift": "^6.0.6",
+    "react-highlight-words": "^0.16.0",
+		"react-use": "^15.3.4"
 	}
 }

